src/wallet/wallet.cpp

```diff
@@ -4010,6 +4007,21 @@ CWallet* CWallet::CreateWalletFromFile(const std::string walletFile)
         }
     }
 
+    walletInstance->m_address_type = ParseOutputType(gArgs.GetArg("-addresstype", ""));
+    if (walletInstance->m_address_type == OutputType::NONE) {
+        InitError(strprintf("Unknown address type '%s'", gArgs.GetArg("-addresstype", "")));
+        return nullptr;
+    }
+
+    // If changetype is set in config file or parameter, check that it's valid.
+    // Default to OutputType::NONE if not set.
+    walletInstance->m_change_type = ParseOutputType(gArgs.GetArg("-changetype", ""), OutputType::NONE);
+    if (walletInstance->m_change_type == OutputType::NONE && !gArgs.GetArg("-changetype", "").empty()) {
+        InitError(strprintf("Unknown change type '%s'", gArgs.GetArg("-changetype", "")));
+        return nullptr;
+    }
+
+
```

Comment:
Nit: unnecessary empty line

Reply:
Sorry. Will keep the white space for now, to not invalidate review.